src/app/components/Figure/Caption/index.jsx

```diff
@@ -1,11 +1,24 @@
 import styled from 'styled-components';
 import { FF_NEWS_SANS_REG, C_STORM } from '../../../lib/constants/styles';
+import mediaQuery from '../../../helpers/mediaQueries';
 
 const Caption = styled.figcaption`
   background-color: #d5d0cd;
   color: ${C_STORM};
   font-family: ${FF_NEWS_SANS_REG};
+  font-size: 0.875em;
+  line-height: 1em;
   padding: 8px;
+  ${mediaQuery.smartPhoneOnly} {
```

Comment:
The `font-size` and `line-height` values at different breakpoints will need to be re-used across different components, e.g. for body copy in paragraphs as well as here in the fig caption. 
The values will also vary for different scripts (Latin, Arabic, etc.). 
I think for these reasons as well as for legibility, it would be better to pull out this logic, have it in the `lib` directory, and require it in as needed. 


For example:
`/lib/typography/news-latin.js`:
```
import mediaQuery from '../helpers/mediaQueries';

export default function typographyLongPrimer() {
  return `
    font-size: 0.875em;
    line-height: 1em;
    ${mediaQuery.smartPhoneOnly} {
      line-height: 1.125em;
    }
    ${mediaQuery.desktopOnly} {
      font-size: 0.813em;
      line-height: 1em;
    }
  `;
}
```

Then in this component file `src/app/components/Figure/Caption/index.jsx`, you have a much easier-to-read component CSS
```
import typographyLongPrimer from '../../../lib/news-latin';

const Caption = styled.figcaption`
  background-color: #d5d0cd;
  color: ${C_STORM};
  font-family: ${FF_NEWS_SANS_REG};
  padding: 8px;
  ${typographyLongPrimer};
`;
```

Reply:
We will park this debate until we have many uses to factor out later. For now I'll comment adding GEL values for clarity.